examples_tests/yarn-with-npm.t

```diff
@@ -1,4 +1,5 @@
-  $ . ${TESTDIR}/setup.sh with-npm yarn
+Ignore Yarn warning that it found a package-lock.json
```

Comment:
you could also delete the packagelock and do a commit like this: 
https://github.com/vercel/turbo/blob/1b383bbb9dcb5b0a203ee55d8fc6510a6123a142/cli/integration_tests/single_package/run-yarn.t#L4-L7